### ISSUE TITLE
'GitLab Commit Issue Publisher' skipped because one of the required properties is missing, while using DotNet

### ISSUE THREAD

**Issue by edcu11** [gabrie-allaigre/sonar-gitlab-plugin#250]
Im using Sonarqube 7.6 with plugin 4.1.0-SNAPSHOT. According to gitlab's pipeline logs, the analysis is successful and I can see it on Sonarqube but no comment is made by the plugin. Im assuming the: 
`'GitLab Commit Issue Publisher' skipped because one of the required properties is missing"`
 has something to do with it. 
Im using dotnet on a Docker image to execute sonar-scan in my yml like this:

` - dotnet sonarscanner begin /k:"monkeyTesting" /d:sonar.login="MY_LOGINKEY" /d:sonar.host.url="MY_SONARKEY:9000"  /d:sonar.verbose=true /d:sonar.cs.opencover.reportsPaths="*.Specs/coverage.opencover.xml" /d:sonar.exclusions="***.Specs/**" /d:sonar.analysis.mode=preview /d:sonar.gitlab.commit_sha=$CI_COMMIT_SHA  /d:sonar.gitlab.ref_name=$CI_COMMIT_REF_NAME /d:sonar.gitlab.project_id=$CI_PROJECT_ID /d:sonar.gitlab.url=https://gitlab.com /d:sonar.issuesReport.console.enable=true`

Ive read some other issues with the same problem (but not with DotNet) and most were solved by adding the "gitlab.url=https://gitlab.com" param, this didnt worked for me. Ive echo the values of my global vars (ci_commit_sha etc) and they are all in check. 
That makes me assume there most be some value Im not passing correctly, given that Im not entirely sure my param names are okay. Since I couldnt find any tutorials or examples about it anywhere. I simply used the same names that are used in the examples for npm, gradle on the repository's readme.

**Comment by lanniu** [comment]
try this "sonar-scanner -X -Dsonar.gitlab.commit_sha=$CI_COMMIT_SHA -Dsonar.gitlab.ref_name=$CI_COMMIT_REF_NAME"

**Comment by edcu11** [comment]
here are MANY params that are needed on the Dotnet sonar scanner that the repo doesnt inform you about:
`/d:sonar.gitlab.user_token=${GITLAB_USER_TOKEN}  /d:sonar.gitlab.url=https://gitlab.com  /d:sonar.gitlab.ignore_certificate=true  /n:sonar.projectName=$PROJECT_KEY /v:sonar.projectVersion=1.0`

User token and gitlab.url weren't included under the needed params at the repo's readme. Project Name and project Version were asked by the scanner itself. I am not sure about the gitlab.ignore_certificate, but I am adding it since I know it solves some other problems. I also removed this  `/d:sonar.issuesReport.console.enable=true`. 

I was also working on a npm project and it worked after I added all those vars except for the version.
This only makes the post - execution job start, I had other problems making the job actually working
